docs/source/index.mdx

```diff
@@ -23,7 +23,8 @@ As such, Optimum enables users to efficiently use any of these platforms with th
 🤗 Optimum aims at providing more diversity towards the kind of hardware users can target to train and finetune their models.
 
 To achieve this, we are collaborating with the following hardware manufacturers in order to provide the best transformers integration:
-- [Graphcore IPUs](https://github.com/huggingface/optimum-graphcore) - IPUs are a completely new kind of massively parallel processor to accelerate machine intelligence. [More information here](https://www.graphcore.ai/products/ipu)
+- [Graphcore IPUs](https://github.com/huggingface/optimum-graphcore) - IPUs are a completely new kind of massively parallel processor to accelerate machine intelligence. [More information here.](https://www.graphcore.ai/products/ipu)
+- [Habana Gaudi Processor (HPU)](https://github.com/huggingface/optimum-habana) - HPUs are designed to maximize training throughput and efficiency. [More information here.](https://habana.ai/training/)
```

Comment:
It could be interesting to give more information on the HPU itself (either with a sentence or a link such as this [one](https://habana.ai/getting-started-with-habana-gaudi-for-deep-learning-training/)), but this can also be done directly in [optimum-habana](https://github.com/huggingface/optimum-habana) so this is only a suggestion !

Reply:
Thanks for the suggestion, I just added a link for HPUs!

Reply:
That looks great, thanks !